pymc4/distributions/continuous.py

```diff
@@ -974,6 +975,64 @@ def _init_distribution(conditions):
         return tfd.LogNormal(loc=loc, scale=scale)
 
 
+class Moyal(ContinuousDistribution):
+    r"""Continuous Moyal random variable.
+
+    The pdf of this distribution is
+
+    .. math::
+
+        f(x \mid \mu, \sigma) = 
+           \frac{1}{\sqrt{2\pi}\sigma}
+           \exp\left(-\frac{1}{2}\left[\frac{x-\mu}{\sigma}+\exp\left(-\frac{x-\mu}{\sigma}\right)\right]\right)
+
+    .. plot::
+
+        import matplotlib.pyplot as plt
+        import numpy as np
+        import scipy.stats as st
+        plt.style.use('arviz-darkgrid')
+        x = np.linspace(-5, 5, 1000)
+        locs = [0., 0., 0., -2.]
+        scales = [0.5, 0.6, 1., 1.]
+        for loc, scale in zip(locs, scales):
+            pdf = st.moyal.pdf(x, loc, scale)
+            plt.plot(x, pdf, label=r'$\mu$ = {}, $\sigma$ = {}'.format(loc, scale))
+        plt.xlabel('x', fontsize=12)
+        plt.ylabel('f(x)', fontsize=12)
+        plt.legend(loc=1)
+        plt.show()
+
+    ========  ==========================================
+    Support   :math:`x \in \mathbb{R}`
+    Mean      :math:`\mu+\sigma(\gamma+ln(2))` where \gamma is the Euler-Mascheroni constant
+    Variance  :math:`\dfrac{\pi^2\sigma^2}{2}`
+    ========  ==========================================
+
+    Parameters
+    ----------
+    loc : float
+        Location parameter.
+    scale : float
+        Scale parameter.
+
+    Examples
+    --------
+    .. code-block:: python
```

Comment:
can you please do something like:
```python
Examples
----------
>>> @pm.model
... def model():
...     x = pm.Moyal('x', loc=0, scale=10)
```

It would be helpful while compiling docs because ``.. code-block::`` directive doesn't work in the Examples section and doc tests fail. Although it is fine as of now (you can make another or for that, if you want to) because all the distributions have this doc style which needs to be fixed.

Reply:
Sure, I have changed the example section as you suggested.